conans/client/build/qbs.py

```diff
@@ -0,0 +1,103 @@
+import os
+
+from conans import tools
+from conans.errors import ConanException
+
+
+class QbsException(ConanException):
```

Comment:
I just defined a custom exception because I wanted to prepend the origin of the exception to the message (qbs build helper and qbs toolchain). Without the custom exception I would need to add this string to every message manually. But if this is not desired I can remove the exceptions.